message_ix/reporting/pyam.py

```diff
@@ -19,10 +20,11 @@ def as_pyam(scenario, year_time_dim, *quantities, drop=[], collapse=None):
     # Convert each of *quantities* individually
     dfs = []
     for qty in quantities:
-        df = qty.to_series() \
-                .rename('value') \
-                .reset_index() \
-                .rename(columns=IAMC_columns)
+        df = qty.to_series() if isinstance(qty, xr.DataArray) else qty
```

Comment:
this should be done using `Quantity` probably

Reply:
Done as iiasa/ixmp@96d98cd.